src/dashboard/cd-dashboard-children.vue

```diff
@@ -155,5 +158,25 @@
       padding: 0 0 0 5px;
     }
   }
+
+  @media (max-width: @screen-xs-max) {
+    .cd-dashboard-children {
+      max-width: 100%;
+
+      &__divider {
+        border-color: #CAC4CE;
+      }
+
+      &__badge {
+        width: 95px;
```

Comment:
The badges text can be fairly long, why flex them rather than that ?

Reply:
Flex them rather than..? Setting the width manually ?

Reply:
*why not flex them rather than having a set width